jupyterlab_gitplus/github_v3.py

```diff
@@ -28,7 +28,7 @@ def create_pull_request(owner_login, repo_name, title, head, base, access_token)
         response.raise_for_status()
         result = {
             'github_url': content['html_url'],
-            'reviewnb_url': content['html_url'].replace('github.com', 'app.reviewnb.com')
+            "reviewnb_url": content['html_url'].replace('https://github.com', reviewnb_endpoint)
```

Comment:
missing a trailing slash in github.com which is creating 2 slashes after the endpoint as shown in screenshot. The line should be 

```
"reviewnb_url": content['html_url'].replace('https://github.com/', reviewnb_endpoint)
```

![Screenshot 2024-06-10 at 7 04 15 PM](https://github.com/ReviewNB/jupyterlab-gitplus/assets/1350490/ecdf254d-2783-4855-9fdb-8a72ab42510c)

Reply:
I've made this fix.

Also, thinking about it, for a user with a custom github domain is likely to have`content["html_url"]` start with `https://github.yourcompany.com" instead of "github.com". Or something like that.

Since I can't test any of this I'll definitely hold off adding the github custom domain functionality.